tests/lib/pkgdb.sh

```diff
@@ -663,6 +663,7 @@ pkg_dependencies_ubuntu_classic(){
                 net-tools
                 packagekit
                 sbuild
+                dbus-user-session
```

Comment:
Please add this as a dependency of snapd as otherwise we are not testing what users are installing.

I'm ok with this being in a follow-up so that we don't have to wait more.

Reply:
I pushed a related change to session-tool-support below